awx/api/renderers.py

```diff
@@ -7,6 +7,24 @@
 # Django REST Framework
 from rest_framework import renderers
 from rest_framework.request import override_method
+from rest_framework.utils import encoders
+
+
+class SurrogateEncoder(encoders.JSONEncoder):
+
+    def encode(self, obj):
+        ret = super(SurrogateEncoder, self).encode(obj)
```

Comment:
This is a little gross, but I couldn't find a less gross way to do this.

tl;dr there’s code further down the stack in DRF that ends up calling `.encode()` (and it blows up), so I’m calling it, seeing if it blows up, and then fixing the problem (so it won’t blow up later)